radio/src/gui/128x64/model_special_functions.cpp

```diff
@@ -139,20 +139,27 @@ void onCustomFunctionsMenu(const char * result)
     storageDirty(eeFlags);
   }
 }
-#endif // CPUARM
+#endif // PCBTARANIS
 
 void menuSpecialFunctions(event_t event, CustomFunctionData * functions, CustomFunctionsContext * functionsContext)
 {
   int8_t sub = menuVerticalPosition - HEADER_LINE;
+#if defined(PCBXLITE)
+  if(menuHorizontalPosition > 4) menuHorizontalPosition = 4;
```

Comment:
Hmm not sure that's the best way, shouldn't it be handled by the menu system? @bsongis